package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cs24-capstone-crowdsourced-video-classification",
   "version": "1.0.0",
   "author": "Jared Beale, Sam Young, Conner Maddalozzo",
-  "description": "",
+  "description": "Web application to crowdsource the effort to associate emotion labels with video clips.",
   "main": "index.js",
   "license": "ISC",
```

Comment:
we should probably change that to a private license found here https://choosealicense.com/licenses/ the ISC is a general license that permites any use of the software AS IS

Reply:
Switched